support-frontend/assets/components/thankYou/ausMap/ausMapItems.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+	LinkButton,
+	SvgArrowRightStraight,
+} from '@guardian/source-react-components';
+import { trackComponentClick } from 'helpers/tracking/behaviour';
+import { OPHAN_COMPONENT_ID_AUS_MAP } from 'pages/contributions-landing/components/ContributionThankYou/utils/ophan';
+
+const AUS_MAP_URL =
```

Comment:
Is is worth making this url relative to the environment we're running in (eg. CODE, local)? Maybe using some of the logic in routes.ts or similar?

Reply:
Yeah this sounds like a good shout! Most of the logic here is ported over from the existing TY page so I'll update this url for the live site too